Dockerfile

```diff
@@ -0,0 +1,24 @@
+ARG NODE_IMAGE_VERSION=13-alpine
```

Comment:
Why? We'll always update the Dockerfile to bump the version, right?

Reply:
This was because of copy pasting. Removed ✅